content/docs/user-guide/experiment-management/index.md

```diff
@@ -47,12 +47,14 @@ The `dvc experiments` features are designed to support these main approaches:
    them in batches.
 1. Make experiments [persistent] by committing them to your
    <abbr>repository</abbr> history.
+1. Easily [share and isolate] experiments using Git and DVC remotes.
```

Comment:
This resolves task #2 from https://github.com/iterative/dvc.org/issues/2687 BTW/

Reply:
"Isolate and share" or "share and isolate"?

Reply:
I like share first since it's the title of the page linked 🙂

Reply:
I guess it could be "and/or" though

Reply:
Hmm actually maybe no need for "isolate" mention here.